packages/features/embed/Embed.tsx

```diff
@@ -321,6 +325,7 @@ const EmailEmbedPreview = ({
   month,
   selectedDateAndTime,
   calLink,
+  userSettingsTimezone,
```

Comment:
Renaming it to `initialTimezone` as component doesn't care from where this value was read from. This is just the initial value.

Reply:
Just a heads-up @hariombalhara , I took over this PR to get it merged because the original contributor did not carry it forward. I no longer have access to make changes to the PR, so you will need to handle it

Reply:
No problem @Amit91848 !! I am already done with the changes. Would appreciate if you can review.

Reply:
Some more things to address. Please wait